R/bootstrap.R

```diff
@@ -500,7 +500,15 @@ tabPanel <- function(title, ..., value = title, icon = NULL) {
 #' @export
 #' @describeIn tabPanel Create a tab panel that drops the title argument.
 #'   This function should be used within `tabsetPanel(type = "hidden")`. See [tabsetPanel()] for example usage.
-tabPanelBody <- function(..., value = NULL, icon = NULL) {
+tabPanelBody <- function(value, ..., icon = NULL) {
+  if (
```

Comment:
Did you decide against moving this check into `tabPanel`?

Reply:
Yes.  It could break others since it is a `stop`. Having an `title = NA` with no value shouldn't prevent users from having the `tabPanel` work.